docs/mautic_community_portal/signing_in_and_configuring_your_profile.rst

```diff
@@ -0,0 +1,251 @@
+
+.. image:: images/cookie-consent.png
+    :alt: VSCode screenshot showing how to change branches
+    :width: 600px
+    :align: center
+
+.. raw:: html
+
+    <br><br>
+
+Setting language preferences
+============================
+
+.. vale off
+
+At the top of the page, you'll notice that there are different languages available if you prefer. Select the language you wish to use and the interface changes into that language. It's worth noting that some aspects of the website may not be fully translated because they're community created, and as a result may only be available in the language they were originally created in. If you would like to help with translating more of the content please post in #translations on :xref:`slack`.
+
+.. vale on
+
+Claiming membership
+*******************
+Once logged in you have the option to claim a Mautic membership.
+
+There is a :xref:`membership-option` as a way to support the Mautic community while also being able to join in with governance processes like voting. You can become an individual member for $100 a year or a pro rated amount based on where you pay taxes, and you can become a corporate member. Read more about the :xref:`membership-option` and explore the perks of corporate membership in this :xref:`corporate-membership`.
+
+If you are already a member of Mautic, you can select this option to match up your membership with your account on the community portal. This allows you to participate in votes, engage in the General Assembly, create proposals, and so forth.
+
+You can watch more about claiming your membership status in this short video:
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/9kOa759Y4eQ" frameborder="0" allowfullscreen></iframe>
+
+.. raw:: html
+
+    <br><br>
+
+
+Mautic maintains a list of who is actually an active member at this point in time, synchronized on a weekly basis.
+
+Unable to claim membership
+==========================
+.. vale off
+
+If you're a member and you're not able to claim your membership as you would expect, please drop a message on Slack in the community channel, because it might be that the email address you used when you paid on Open Collective doesn't actually match the email address that you use on the community portal.
+
+Some manual tweaks can fix this quickly, so drop a message in the channel and the team can take a look.
+
+.. vale on
+
+Managing your account
+**********************
+Check the dropdown menu item underneath your name. This allows you to customize your account, your public profile, and also see notifications and direct messages. If you're an administrator, you access the administrator portal here, too.
```

Comment:
```suggestion
Check the dropdown menu item underneath your name. This allows you to customize your account, your public profile, your timezone, and also see notifications and direct messages. 
```

Admins access via a yellow bar at the top of the page now, updating to reflect that.